### ISSUE TITLE
Built-in leave-one-out validation in Keras

### ISSUE THREAD

**Issue by ZsocaCoder** [fchollet/keras#4891]
Hello everyone!

Could anyone tell me, please, whether a built-in leave-one-out validation exists in Keras? I mean, let's assume I have only a training data set consisting of 100 records and want to build the model 100 times based on 99 records and validating it on the remaining one. Would like to avoid implementing this on my own with loops, if possible. :)
Of course, during this process I'd like to accumulate the error of predictions (basically, my target variable is continuous, so speeking here about a regression task) from model to model.

Thank you very much 4 your responses in advance!

Almost forgot: Happy New Year! :)

**Comment by ohadle** [comment]
I think you could use the [sklearn wrappers](https://keras.io/scikit-learn-api/) and take advantage of sklearn's [awesome CV tools](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.LeaveOneOut.html).

**Comment by ZsocaCoder** [comment]
Thank you, ohadle, will take a look soon!

**Comment by ZsocaCoder** [comment]
AFAIK, there aren't built-in features for cross-validation in Keras but there are other libraries that ohadle pointed out. Anyway, in order to avoid randomness I also managed to implement it with simple loop statements.

**Comment by NoraAMM** [comment]
> AFAIK, there aren't built-in features for cross-validation in Keras but there are other libraries that ohadle pointed out. Anyway, in order to avoid randomness I also managed to implement it with simple loop statements.

Could you show us the code?
